docs/user-guide/leverage-cli/reference/shell.md

```diff
@@ -0,0 +1,28 @@
+# Command: `shell`
+
+Run a shell in a generic container.
+It supports mounting local paths and injecting arbitrary environment variables.
+It also supports AWS credentials injection via mfa/sso.
+
+```bash
+>> leverage shell --help
+
+Usage: leverage shell [OPTIONS]
+
+  Run a shell in a generic container. It supports mounting local paths and
+  injecting arbitrary environment variables. It also supports AWS credentials
+  injection via mfa/sso.
+
+  Syntax: leverage shell --mount <local-path> <container-path> --env-var <name> <value>
+  Example: leverage shell --mount /home/user/bin/ /usr/bin/ --env-var env dev
+
+  Both mount and env-var parameters can be provided multiple times.
+  Example: leverage shell --mount /home/user/bin/ /usr/bin/ --mount /etc/config.ini /etc/config.ini --env-var init 5 --env-var env dev
+
+Options:
+  --mount <TEXT TEXT>...
```

Comment:
```suggestion
  --mount <HOST_PATH> <CONTAINER_PATH>...
```
Is the above supported? Does it need to use quotes?

Reply:
no quotes, is the way Click show tuples unfortunately

We can change the documentation but `leverage shell --help` will still show it that way.
I think the clarification here: https://github.com/binbashar/le-ref-architecture-doc/pull/205/files#diff-de2efff7f5cb5f149039d444f61b11b87794aca85cd4664f5cfbcf582f180ac0R16
and the example below might be enough to clarify

Reply:
That's ok. It looked strange to me to that it shows "<TEXT TEXT>" but the examples right above show clearly how to use the command.